RoboticonColony/Assets/Code/Tile.cs

```diff
@@ -0,0 +1,72 @@
+﻿using UnityEngine;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+sealed public class Tile
+{
+    public int Cost { get; private set; }
+    public int OwnerID { get; private set; }
+
+    private Dictionary<ItemType, int> _Resources;
+    private int _ownerID;
+
+    /// <summary>
+    /// Constructor for tile
+    /// </summary> 
+    /// <param name="cost">Cost of the tile</param>
+    /// <param name="ore">Ore that the tile has</param>
+    /// <param name="power">Power that the tile has</param>
+    public Tile(int cost, int ore, int power)
+    {
+        Cost = cost;
+        _Resources = new Dictionary<ItemType, int>();
+        _Resources[ItemType.Ore] = ore;
+        _Resources[ItemType.Power] = power;
+        OwnerID = -1;
+    }
+
+    /// <summary>
+    /// Adds a roboticon to the tile
+    /// </summary>
+    /// <param name="newRoboticon">Roboticon to be added</param>
+    //public void AddRoboticon(Roboticon newRoboticon)
+    //{
+    //    _Resources[ItemType.Roboticon] = newRoboticon;
+    //}
+
+    /// <summary>
+    /// Removes the roboticon from the tile
+    /// </summary>
+    //public void RemoveRoboticon()
+    //{
+    //    _Resources[ItemType.Roboticon] = null;
+    //}
+
+    /// <summary>
+    /// Adds the resources on the tile to the player's inventory
+    /// </summary>
+    public void Produce(Inventory playerInv)
+    {
+        playerInv.AddItem(ItemType.Ore, Ore);
+        playerInv.AddItem(ItemType.Power, Power);
+    }
+
+    public int Ore
```

Comment:
Better to merge these?

public int GetResource(ItemType resourceType){
         return _Resources[resourceType];
}